src/main/kotlin/no/nav/meldeplikt/meldekortservice/api/Meldekort.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.meldeplikt.meldekortservice.api
+
+import io.ktor.application.ApplicationCallPipeline
+import io.ktor.application.call
+import io.ktor.client.HttpClient
+import io.ktor.http.ContentType
+import io.ktor.http.HttpStatusCode
+import io.ktor.response.respond
+import io.ktor.response.respondText
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import io.ktor.routing.route
+
```

Comment:
Det jeg mente var f.eks noe slik:
/**
 * REST-controller for meldekort API som tilbyr operasjoner for å hente meldekortdetaljer og korrigering for en NAV-bruker.
 */